apps/prairielearn/src/models/course-instance-publishing-extensions.sql

```diff
@@ -0,0 +1,97 @@
+-- BLOCK select_publishing_extensions_by_enrollment_id
+SELECT
+  ci_extensions.*
+FROM
+  course_instance_publishing_extensions AS ci_extensions
+  JOIN course_instance_publishing_enrollment_extensions AS ci_enrollment_extensions ON (
+    ci_enrollment_extensions.course_instance_publishing_extension_id = ci_extensions.id
+  )
+WHERE
+  ci_enrollment_extensions.enrollment_id = $enrollment_id;
+
+-- BLOCK select_publishing_extension_by_name
+SELECT
+  *
+FROM
+  course_instance_publishing_extensions
+WHERE
+  course_instance_id = $course_instance_id
+  AND name = $name;
```

Comment:
AFAICT this and other queries here are never actually used on this branch. I'm assuming they will be on the UI one?